pipelines/tekton/build-container-image-pipeline/check-model-and-containerfile-exists.yaml

```diff
@@ -12,14 +12,16 @@ spec:
     type: string
   - name: containerfilePath
     type: string
+  - name: modelRelativePath
+    type: string
   steps:
   - name: check-model-and-containerfile
     image: registry.access.redhat.com/ubi9/ubi-micro
     script: |
       #!/usr/bin/env bash
 
       # Check model files exist
-      ls -l $(params.model-name)/
+      ls -l model_dir/$(params.modelRelativePath)/$(params.model-name)/
```

Comment:
Something to keep in mind for future: we should decide on which casing we want to use for params here, and use that consistently (i.e. paramName vs param-name vs other varieties).

Reply:
Yeah I agree, I mentioned this is Srijoni's PR. I didn't want to go changing other things though 😄